src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1139,7 +1143,7 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                             case AttachmentReplicationItem attachment:
 
                                 var localAttachment = database.DocumentsStorage.AttachmentsStorage.GetAttachmentByKey(context, attachment.Key);
-                                if (_replicationInfo.ReplicatedAttachmentStreams.TryGetValue(attachment.Base64Hash, out var attachmentStream))
+                                if (_replicationInfo.ReplicatedAttachmentStreams?.TryGetValue(attachment.Base64Hash, out var attachmentStream) == true)
```

Comment:
This worries me a bit, because we now may send it with no values (in fact, that was already the case previously, mind), which can be problematic.